src/track/track.h

```diff
@@ -245,6 +245,8 @@ class Track : public QObject {
     CuePosition getCuePoint() const;
     // Set the track's main cue point
     void setCuePoint(CuePosition cue);
+    /// Shift all cues by a constant offset
+    void shiftCuePositions(double milliseconds);
```

Comment:
Renaming suggested: shiftCuePositionsMillis (because *cue positions* are still measured in *stereo samples* if not denoted otherwise)